config/package/resources.yaml.gotmpl

```diff
@@ -95,7 +96,7 @@ spec:
         - -cacert
         - /service-ca/service-ca.crt
         - -tls
-        image: REPLACED_BY_PIPELINE
+        image: quay.io/app-sre/managed-cluster-validating-webhooks:4c7a8ed
```

Comment:
We shouldn't need to change this line, it gets changed during the app interface CI build to reflect the latest commit.